App/navigators/RootStack.tsx

```diff
@@ -37,6 +59,72 @@ const RootStack: React.FC = () => {
     navigation.navigate(Screens.Terms)
   }
 
+  const initAgent = async () => {
+    if (initAgentInProcess) {
+      return
+    }
+
+    Toast.show({
+      type: ToastType.Info,
+      text1: t('StatusMessages.InitAgent'),
+      position: 'bottom',
+    })
+
+    //Flag to protect the init process from being duplicated
+    setInitAgentInProcess(true)
+
+    //Remove before production
```

Comment:
I don't quite understand this comment--is this an outdated comment?

Reply:
"//Remove before production" should be removed yes